instrumentation/opentelemetry_ecto/lib/opentelemetry_ecto.ex

```diff
@@ -201,6 +205,14 @@ defmodule OpentelemetryEcto do
     end)
   end
 
+  defp maybe_add_db_operation(attributes, nil) do
+    attributes
+  end
+
+  defp maybe_add_db_operation(attributes, db_operation) do
+    Map.put(attributes, Trace.db_operation(), db_operation)
```

Comment:
This is not a valid attribute any longer so this would need to be introduced with semconv 1.26. It's also experimental so it'll have to be opt-in.

For `db.operation.name` the value is supposed to be the command or operation being invoked (function name, SQL operation, etc), so I'm not sure it can be changed by the user.

> [5]: It is RECOMMENDED to capture the value as provided by the application without attempting to do any case normalization. If the operation name is parsed from the query text, it SHOULD be the first operation name found in the query. For batch operations, if the individual operations are known to have the same operation name then that operation name SHOULD be used prepended by BATCH , otherwise db.operation.name SHOULD be BATCH or some other database system specific term if more applicable.

Reply:
Should I hard-code it for now without using `Trace.*`? I intended to use SemConv. I wasn't sure what to do about it.

> For db.operation.name the value is supposed to be the command or operation being invoked (function name, SQL operation, etc), so I'm not sure it can be changed by the user.

I asked for direction in the Slack channel, the wording and examples are confusing me!

Reply:
@yordis you can look at `opentelemetry_xandra` where I use all hardcoded semcov names exactly because we are waiting for 1.26

Reply:
Either way it is an experimental attribute which requires part of the rewrite of accepted options and overall processing. I'm happy to work through what the values can be so that it can go in there but it will not be published in a release before then.

Reply:
That works for me!

Based on the description, it could be anything from the app, so I think my intent is correct.

Maybe we should follow up with OTEL folks; regardless, I would love to have a mechanism to have an identity for the operation.

Maybe I should expand the feature to allow passing of any value into the span `telemetry_options: [otel_attributes: [...]] (to make a point)